client/src/containers/buildLog/buildLog.tsx

```diff
@@ -1,17 +1,23 @@
 import React, { useEffect } from 'react';
-import PropTypes from 'prop-types';
 import { useDispatch, useSelector } from 'react-redux';
 import clsx from 'clsx';
 import { cn } from 'utils/bem';
 
 import { getBuildLog } from 'store/builds/getBuildLog';
+import { State } from 'store/store';
+import BuildLogModel from 'typings/buildLog';
 
 import './buildLog.css';
 
-const BuildLog = props => {
+export interface BuildLogProps {
+  className?: string;
+  buildId: string;
+}
+
+const BuildLog: React.FC<BuildLogProps> = props => {
   const { buildId, className = '' } = props;
   const dispatch = useDispatch();
-  const log = useSelector(state => state.builds.log);
+  const log = useSelector((state: State): BuildLogModel => state.builds.log);
```

Comment:
```suggestion
  const log = useSelector((state: State) => state.builds.log);
```

 В доке https://redux.js.org/recipes/usage-with-typescript есть пример как немного подсказать компилятору чтобы он получше выводил типы:
```
export default (state = initState, action: BuildsActions): BuildsState => {
```

Reply:
Так действительно гораздо лучше, исправила, спасибо!